test/routing_test.rb

```diff
@@ -630,17 +630,26 @@ class RoutingTest < Minitest::Test
   end
 
   it 'makes regular expression captures available in params[:captures] for concatenated routes' do
-    concatenated = Mustermann.new('/prefix') + Mustermann.new("/fo(.*)/ba(.*)", :type => :regexp)
+    with_regexp = Mustermann.new('/prefix') + Mustermann.new("/fo(.*)/ba(.*)", :type => :regexp)
+    without_regexp = Mustermann.new('/prefix') + Mustermann.new('/baz')
```

Comment:
Maybe this is not `Mustermann::Concat`, pure `Mustermann::Sinatra`.
You should use multiple mustermann types, it will be something like:

```ruby
without_regexp = Mustermann.new('/prefix', type: :identity) + Mustermann.new('/baz')
```